src/components/calculators/Payslip.vue

```diff
@@ -111,7 +111,7 @@ Payslip calculator.
       <template v-if="payslipResult">
         <p class="pw-calc-output__description">Results based on <strong>{{ resultPeriod }}</strong> Gross pay of <span>{{ resultGrossPay | currency }}</span></p>
       </template>
-      <tabs>
+      <tabs @clicked="tabClicked">
```

Comment:
Was just `@clicked` enough?
According to the docs 
```
changed is emitted when the tab changes and can be used as handle to load data on request. clicked is emitted when an active tab is re-clicked and can be used to e.g. reload the data in the current tab.
```
So that suggests that clicked only fires if you click the tab that you're already on.

Reply:
@david-robinson-practiceweb  @clicked seemed to be enough it's working just like I demo'd previously and it's still working like that now. Just with @clicked.

Would you recommend applying both ?